doc/source/train/examples/deepspeed/gptj_deepspeed_fine_tuning.ipynb

```diff
@@ -1406,10 +1413,10 @@
    },
    "outputs": [],
    "source": [
-    "!awsv2 configure set s3.max_concurrent_requests 32\n",
-    "!awsv2 configure set default.s3.preferred_transfer_client crt\n",
-    "!awsv2 configure set default.s3.target_bandwidth 100Gb/s\n",
-    "!awsv2 configure set default.s3.multipart_chunksize 8MB"
+    "!aws configure set s3.max_concurrent_requests 32\n",
+    "!aws configure set default.s3.preferred_transfer_client crt\n",
+    "!aws configure set default.s3.target_bandwidth 100Gb/s\n",
+    "!aws configure set default.s3.multipart_chunksize 8MB"
```

Comment:
these only work on `awsv2` to make syncing down faster. We can remove this cell, but need to make sure the tests don't time out due to the slower download.

Reply:
aws is v2 now.

Reply:
it used to v1, because we did not install v2.

Reply:
if you want to enforce v2, you should run `aws --version` and check its output.

Reply:
`awsv2` is a hobbyist project on pypi by an individual not by aws:
https://pypi.org/project/awscliv2/

we have no good reason to use it here.